web/src/frontend/scripts/mock_data/fakta.json

```diff
@@ -10,58 +10,258 @@
 		"type": "BRUKERREGISTRERT"
 	},
 	{
-		"faktumId": 5483,
-		"soknadId": 88,
+		"faktumId": 2,
+		"soknadId": 1,
 		"parrentFaktum": null,
 		"key": "personalia",
 		"value": null,
+		"faktumEgenskaper": [
+			{
+				"faktumId": 2,
+				"soknadId": 1,
+				"key": "sekundarAdresseGyldigTil",
+				"value": "2012-11-12",
+				"systemEgenskap": 1
+			},
+			{
+				"faktumId": 2,
+				"soknadId": 1,
+				"key": "statsborgerskap",
+				"value": "NOR",
+				"systemEgenskap": 1
+			},
+			{
+				"faktumId": 2,
+				"soknadId": 1,
+				"key": "sekundarAdresseGyldigFra",
+				"value": "2017-09-06",
+				"systemEgenskap": 1
+			},
+			{
+				"faktumId": 2,
+				"soknadId": 1,
+				"key": "gjeldendeAdresseGyldigFra",
+				"value": null,
+				"systemEgenskap": 1
+			},
+			{
+				"faktumId": 2,
+				"soknadId": 1,
+				"key": "navn",
+				"value": "Donald D. Mockmann",
+				"systemEgenskap": 1
+			},
+			{
+				"faktumId": 2,
+				"soknadId": 1,
+				"key": "kontonummer",
+				"value": "9876 98 98765",
+				"systemEgenskap": 1
+			},
+			{
+				"faktumId": 2,
+				"soknadId": 1,
+				"key": "kjonn",
+				"value": "m",
+				"systemEgenskap": 1
+			},
+			{
+				"faktumId": 2,
+				"soknadId": 1,
+				"key": "sekundarAdresse",
+				"value": "Poitigatan 55, Nord-Poiti, 1111 Helsinki, Finland, Finland",
+				"systemEgenskap": 1
+			},
+			{
+				"faktumId": 2,
+				"soknadId": 1,
+				"key": "statsborgerskapType",
+				"value": "norsk",
+				"systemEgenskap": 1
+			},
+			{
+				"faktumId": 2,
+				"soknadId": 1,
+				"key": "fnr",
+				"value": "03076321565",
```

Comment:
Er dette reelt fnr?

Reply:
Jeg vet ikke. Dette er fnr for Donald D. Mockmann.

Reply:
Jeg tror det er dette som var det som Mats fjernet en gang tidligere